arcusApp/app/src/main/java/arcus/app/device/adapter/DeviceDetailPagerAdapter.java

```diff
@@ -185,15 +184,6 @@ private Fragment getFragmentForDevice(@NonNull DeviceModel device) {
             case SMOKE:
                 product = SmokeDetectorFragment.newInstance(false);
                 break;
-            case THERMOSTAT:
```

Comment:
Perhaps there's a bug on line 189 where Thermostat uses NestThermostatFragment? probably need this code